plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -25,7 +25,7 @@
   agent:
     description:
       - Specify if the QEMU Guest Agent should be enabled/disabled.
```

Comment:
```suggestion
      - Specify if the QEMU Guest Agent should be enabled/disabled.
      - Since community.general 5.5.0, this can also be a string instead of a boolean.
        This allows to specify values such as C(enabled=1,fstrim_cloned_disks=1).
```